src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js

```diff
@@ -208,41 +208,33 @@ describe('signals/incident-management/components/FilterForm', () => {
     const nameField = container.querySelector('input[type="text"][name="name"]');
     const dateField = container.querySelector('input[id="filter_created_before"]');
     const addressField = container.querySelector('input[type="text"][name="address_text"]');
+    const noteField = container.querySelector('input[type="text"][name="note_keyword"]');
     const afvalToggle = container.querySelector('input[type="checkbox"][value="afval"]');
 
-    act(() => {
-      fireEvent.change(nameField, { target: { value: 'My filter' } });
-    });
-    act(() => {
-      fireEvent.change(dateField, { target: { value: '1970-01-01' } });
-    });
-    act(() => {
-      fireEvent.change(addressField, {
-        target: { value: 'Weesperstraat 113' },
-      });
-    });
-    act(() => {
-      fireEvent.click(afvalToggle, new MouseEvent({ bubbles: true }));
-    });
+    act(() => { fireEvent.change(nameField, { target: { value: 'My filter' } }); });
+    act(() => { fireEvent.change(dateField, { target: { value: '1970-01-01' } }); });
+    act(() => { fireEvent.change(addressField, { target: { value: 'Weesperstraat 113' } }); });
+    act(() => { fireEvent.change(noteField, { target: { value: 'test123' } }); });
+    act(() => { fireEvent.click(afvalToggle, new MouseEvent({ bubbles: true })); });
 
     wait(() => {
       expect(nameField.value).toEqual('My filter');
       expect(dateField.value).toEqual('1970-01-01');
       expect(addressField.value).not.toBeFalsy();
+      expect(noteField.value).toEqual('test123');
       expect(afvalToggle.checked).toEqual(true);
       expect(container.querySelectorAll('input[type"checkbox"]:checked').length).toBeGreaterThan(1);
     });
 
-    act(() => {
-      fireEvent.click(container.querySelector('button[type="reset"]'));
-    });
+    act(() => { fireEvent.click(container.querySelector('button[type="reset"]')); });
 
     wait(() => {
       expect(onClearFilter).toHaveBeenCalled();
 
       expect(nameField.value).toEqual('');
       expect(dateField.value).toEqual('');
       expect(addressField.value).toEqual('');
+      expect(noteField.value).toEqual('');
       expect(afvalToggle.checked).toEqual(false);
       expect(container.querySelectorAll('input[type"checkbox"]:checked').length).toEqual(0);
```

Comment:
Not part of the changes, but the selector should be `input[type="checkbox"]:checked`

Reply:
Will fix ;)